src/content/en/updates/posts/2016/03/mse-sourcebuffer.markdown

```diff
@@ -0,0 +1,109 @@
+---
+layout: updates/post
+title: "Media Source API: Automatically Ensure Seamless Playback of Media Segments in Append Order"
+description: "The Media Source API enables JavaScript to construct media streams for playback. From Chrome 50, it's possible to use <code>SourceBuffer</code> <em>sequence</em> mode to ensure media segments are automatically relocated in the timeline in the order they were appended, without gaps between them."
+published_on: 2016-03-10
+updated_on: 2016-03-10
+authors:
+  - samdutton
+tags:
+  - audio
+  - chrome50
+  - media
+  - mse
+  - news
+  - recording
+  - video
+  - webrtc
+  - chrome50
+featured_image: /web/updates/images/2016/03/mse-sourcebuffer/featured.jpg
+---
+
+<style>
+@media screen and (max-width: 500px) {
+  img.screenshot {
+    max-width: 100%;
+  }
+}
+</style>
+
+<p class="intro">The HTML audio and video elements enable you to load, decode and play media, simply by providing a src URL:
+</p>
+
+{% highlight html %}
+<video src='foo.webm'></video>
+{% endhighlight %}
+
+That works well in simple use cases, but for techniques such as [adaptive streaming](https://www.youtube.com/watch?v=Fm3Bagcf9Oo), the Media Source Extensions API (MSE) provides more control. MSE enables streams to be built in JavaScript from segments of audio or video.
+
+You can try out MSE at [simpl.info/mse](https://simpl.info/mse):
```

Comment:
I'd like to do that, but I don't think I'll have time today. Agree it's not ideal linking to external samples, but at least it has a nice short URL...